<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_searchSong">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>노래 탐색</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>11</y>
     <width>371</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_YtbOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>재생</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxYtbOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPage_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dcincide">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>원본 페이지</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_RandYtbOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>랜덤재생</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Nowlisten">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>109</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_play">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>재생중</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
